src/main/java/hudson/plugins/promoted_builds/conditions/ManualCondition.java

```diff
@@ -59,6 +59,10 @@ public String getUsers() {
         return users;
     }
 
+    public void setUsers(String users) {
```

Comment:
Wrong indenting (use spaces)
It also needs Javadocs with <code>@since</code> specification